backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphics.java

```diff
@@ -79,7 +80,7 @@
 	}
 
 	LwjglGraphics (Canvas canvas, LwjglApplicationConfiguration config) {
-		this.config = config;
+		this(config);
```

Comment:
@3xp0n3nt I know that it doesn't matter. I also know this pattern, but people normally have a reason for a change :)